modules/openideal_rest/openideal_rest.module

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * @file
+ * Contains main functionality/hooks for openideal_rest module.
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\openideal_rest\WebhooksFormProcessor;
+
+/**
+ * Implements hook_webhooks_event_info_alter().
+ */
+function openideal_rest_webhooks_event_info_alter(&$options) {
+  $options = [];
+
+  /** @var \Drupal\message\Entity\MessageTemplate[] $message_templates */
+  $message_templates = \Drupal::entityTypeManager()->getStorage('message_template')->loadMultiple();
+
+  foreach ($message_templates as $message) {
+    $options[$message->id()] = [
+      'type' => $message->label(),
+      'event' => $message->id(),
+    ];
+  }
+}
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function openideal_rest_form_webhooks_settings_alter(&$form, FormStateInterface $form_state) {
+  \Drupal::classResolver(WebhooksFormProcessor::class)
+    ->alter($form, $form_state);
+}
+
+/**
+ * Implements hook_form_BASE_FORM_ID_alter().
+ *
+ * Adds additional "plugin" field into the webhook form.
+ */
+function openideal_rest_form_webhook_config_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  $entity = $form_state->getFormObject()->getEntity();
+
+  if (isset($form['label'])) {
+    $form['label']['#weight'] = -3;
+  }
+
+  if ($form['type']) {
```

Comment:
Let's be consistent with previous section and add `isset`.

Reply:
Removed this part totally.